components/Breadcrumb/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@gemeente-denhaag/breadcrumb",
+  "description": "A Breadcrumb component",
+  "author": "Municipality of The Hague",
+  "license": "EUPL-1.2",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "exports": "./dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup -c ../../rollup.config.js",
+    "clean": "yarn rimraf dist tsconfig.tsbuildinfo"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nl-design-system/denhaag.git",
+    "directory": "components/Breadcrumb"
+  },
+  "bugs": "https://github.com/nl-design-system/denhaag/issues",
+  "dependencies": {
+    "@gemeente-denhaag/baseprops": "0.2.3-alpha.63"
```

Comment:
This component does not use the `baseprops`

Reply:
removed it.